network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -110,12 +111,21 @@ def is_chapter_page(self):
         parent = self.get_parent().specific
         return parent.__class__ is PublicationPage
 
+    @property
+    def zen_nav(self):
+        return True
```

Comment:
Q: How is this different than just setting `zen_nav = True` <--- which is what we've been doing

Reply:
Currently, nothing. It's just overwrite-able through class inheritance. Do we prefer `zen_nv = True` as a concrete property?

Reply:
No pref here. Marking this as resolved!